requirements.txt

```diff
@@ -15,20 +15,25 @@ cachetools==4.2.2
 certifi==2021.5.30
 cffi==1.14.5
 chardet==3.0.4
+click==8.0.1
```

Comment:
your requirements.txt looks very bloated -- most of these libraries are not needed. did you put all of your locally installed libraries in here?